modules/versions.tf

```
@@ -0,0 +1,11 @@
+terraform {
+
+  required_version = "~> 1.2.3"
```

```suggestion
  required_version = "~> 1.3.6"
```
Can we upgrade to 1.3.X?